File: PostBox/src/main/java/com/mvp/view/PostOffice.java
package com.mvp.view;

import java.util.Scanner;

public class PostOffice {

    public static void main(String[] args) {
        System.out.println("Hi, please enter your box for validating");
        System.out.println("first enter the length of box : ");
        Scanner scanner = new Scanner(System.in);
        float length = scanner.nextFloat();
        System.out.println("then enter the width of box : ");
        float width = scanner.nextFloat();
        System.out.println("finally enter the height of box :");
        float height = scanner.nextFloat();

        // create presenter instance in a custom view
        Window postOfficeWindow = new Window();
        postOfficeWindow.receivedBox(length, width, height);
    }
}

File: PostBox/src/main/kotlin/com/mvp/view/PostOffice.kt
package com.mvp.view.kt

fun main() {
    enterPostOffice()
}

private fun enterPostOffice() {
    println("Hi, please enter your box size for validating")
    println("first, enter the length of box : ")
    val length = readLine()?.toFloat()
    println("then enter the width of box : ")
    val width = readLine()?.toFloat()
    println("finally enter the height of box :")
    val height = readLine()?.toFloat()

    val businessWindow = BusinessWindow()
    businessWindow.receivedBox(length, width, height)
}